pipdeptree.py

```diff
@@ -548,10 +557,7 @@ def get_parser():
     return parser
 
 
-def main():
-    parser = get_parser()
-    args = parser.parse_args()
-
+def main(args):
```

Comment:
This change will break installation of the pkg because `pipdeptree:main` is specified as the console script entry point in `setup.py`. For ease of testing you can either define another intermediate function (say `run`) or use mocks.